### ISSUE TITLE
Which ViT weights did you originally use in the paper?

### ISSUE THREAD

**Issue by XiongweiWu** [rstrudel/segmenter#7]
Hi, I see some improvements of your released model compared with the results reported in your paper (51 vs 48 for Seg/L-16, etc.), and you mention this performance gain is from the improved ViT weights. I wonder which ViT weights did you use in your paper?

**Comment by rstrudel** [comment]
Hi @XiongweiWu ,
Indeed, with the recently released ViT weights and an improved mask transformer, we trained Segmenter models with better performance, especially Seg-L-Mask/16. We plan on updating the paper to be consistent with the new reported results.

Related to our decoder, we modified the mask transformer by adding a L2 normalization before the mask generation and using a layer normalization on the generated masks which resulted in a more stable training and improved performance.

Related to ViT, we observed a correlation between the top-1 accuracy on ImageNet and mean IoU. For example, the previous publicly available ViT-L model had a 85% top-1 accurracy while the improved ViT-L model reaches 87%. We used the 87% checkpoint to initialize the weights of Seg-L-Mask/16 backbone and obtained a mean IoU of 51.3% to be compared with the 48.8% of the paper as you metionned. We also trained Seg-L-Mask/16 with a resolution of 640x640, matching Swin-Transformers resolution, and report 53.6% mIoU with multi-scale inference.

Originally we used [ViT](https://arxiv.org/abs/2010.11929) and [DeiT](https://arxiv.org/abs/2012.12877) weights which are available in [timm](https://github.com/rwightman/pytorch-image-models) 0.4.8 . Now we are using the updated weights available in timm 0.4.12 . I hope this answers your question.
